### DISCUSSION TITLE
[FEATURE] New challenge "Colorful Message"

### DISCUSSION THREAD

**Discussion by kittens-code** [lennis-dev/hackit#14] [Ideas]
### Is there an existing discussion for this?

- [X] I have searched the existing discussions

### Describe your idea

I would like to add a new challenge, where you work with colors.

Name: Colorful Message

Description: You know, sometimes, you gotta look behind the colors to find out the solution...

**Comment by lennis-dev** [comment]
Nice idea, I convert this to an issue
